Travel Assist APP

grmurad opened the pull request:
This is my contribution for Travel app lab part of SuperDataScience program.

grmurad committed:
Created using Colab

grmurad committed:
Criado usando o Colab

grmurad committed:
Criado usando o Colab

grmurad committed:
Criado usando o Colab; added routine to setup files upon start of google colab session

grmurad committed:
Add files via upload

Those are HTML and data files used for the main Flask/ LLM application

grmurad committed:
Criado usando o Colab

grmurad committed:
Criado usando o Colab

grmurad committed:
Update README.md

Added content for explaining solution from Gustavo Murad

grmurad committed:
Criado usando o Colab

grmurad committed:
Criado usando o Colab

grmurad committed:
Fix handling google maps error when long lat not present in hotel data

grmurad committed:
Ajuste logs

grmurad committed:
version to cleanup origin city if users enter accent like ~, ç, é, etc...which was damaging the IATA search and adding aditional step in the data entry.

grmurad committed:
Added more attractive places for more diverse background pictures

grmurad committed:
Adjust responsive setup for better mobile usability

SuperDataScience-Community commented:
Hi @grmurad I can see that there are a lot of files that are being pushed to the root of the project folder. Could you please create a folder inside of the notebooks/ folder with your name and place all the files in that folder?

Eg. notebooks/gustavo/

grmurad commented:
Hi!! Thats what I wanted but was not being able to. Will retry.

Tks!

Enviado do Gmail para celular


Em ter., 18 de mar. de 2025 às 16:46, SuperDataScience-Community <
***@***.***> escreveu:

> Hi @grmurad <https://github.com/grmurad> I can see that there are a lot
> of files that are being pushed to the root of the project folder. Could you
> please create a folder inside of the notebooks/ folder with your name and
> place all the files in that folder?
>
> Eg. notebooks/gustavo/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14#issuecomment-2734535048>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANIDH2ODBQKXU54QP73FWNT2VBZ2NAVCNFSM6AAAAABZCYFPUGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZUGUZTKMBUHA>
> .
> You are receiving this because you were mentioned.Message ID:
> <SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14/c2734535048
> @github.com>
> [image: SuperDataScience-Community]*SuperDataScience-Community* left a
> comment
> (SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion#14)
> <https://github.com/SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14#issuecomment-2734535048>
>
> Hi @grmurad <https://github.com/grmurad> I can see that there are a lot
> of files that are being pushed to the root of the project folder. Could you
> please create a folder inside of the notebooks/ folder with your name and
> place all the files in that folder?
>
> Eg. notebooks/gustavo/
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14#issuecomment-2734535048>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANIDH2ODBQKXU54QP73FWNT2VBZ2NAVCNFSM6AAAAABZCYFPUGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDOMZUGUZTKMBUHA>
> .
> You are receiving this because you were mentioned.Message ID:
> <SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14/c2734535048
> @github.com>
>

grmurad committed:
Create readme

create directory for contribute

grmurad committed:
Add files via upload

Those are files required for the app to run: 
- HTML for UI input and presentations of results
- CSV for city/location handling
- JSON for city/IATA

grmurad committed:
Add files via upload

LAtest version main app/code flask AssistTravel

grmurad committed:
Delete index.html

grmurad committed:
Delete results.html

grmurad committed:
Delete selecionar_cidade.html

grmurad committed:
Delete cidades_filtradas.json

grmurad committed:
Delete IATA_Cities.csv

grmurad committed:
Delete HI9M.gif

grmurad committed:
Criado usando o Colab

grmurad committed:
Criado usando o Colab

grmurad committed:
Delete " directory

grmurad committed:
Criado usando o Colab

grmurad committed:
Add files via upload

grmurad committed:
Delete Gustavo Murad/Assisttravel.ipynb

delete

grmurad committed:
Rename Assisttravel (1).ipynb to Assisttravel.ipynb

grmurad committed:
Delete Gustavo Murad/results.html

delete

grmurad committed:
Add files via upload

grmurad committed:
Delete Assisttravel.ipynb

grmurad commented:
hi, now tried to pack it into a single folder.

SuperDataScience-Community-User commented:
Hi @grmurad I have merged your code in but I have removed the section from the Readme where you mentioned details about your project specifically. The main README should not be changed as that is the typical README for the entire project.

You can include a README inside of your folder with the specific details you want to mention about your project.

grmurad commented:
Understood! Tks

Enviado do Gmail para celular


Em qui., 27 de mar. de 2025 às 09:09, SuperDataScience-Community-User <
***@***.***> escreveu:

> Hi @grmurad <https://github.com/grmurad> I have merged your code in but I
> have removed the section from the Readme where you mentioned details about
> your project specifically. The main README should not be changed as that is
> the typical README for the entire project.
>
> You can include a README inside of your folder with the specific details
> you want to mention about your project.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14#issuecomment-2757811807>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANIDH2KJCSVCBQXU6WFZWKL2WPTANAVCNFSM6AAAAABZCYFPUGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONJXHAYTCOBQG4>
> .
> You are receiving this because you were mentioned.Message ID:
> <SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14/c2757811807
> @github.com>
> [image: SuperDataScience-Community-User]*SuperDataScience-Community-User*
> left a comment
> (SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion#14)
> <https://github.com/SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14#issuecomment-2757811807>
>
> Hi @grmurad <https://github.com/grmurad> I have merged your code in but I
> have removed the section from the Readme where you mentioned details about
> your project specifically. The main README should not be changed as that is
> the typical README for the entire project.
>
> You can include a README inside of your folder with the specific details
> you want to mention about your project.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14#issuecomment-2757811807>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANIDH2KJCSVCBQXU6WFZWKL2WPTANAVCNFSM6AAAAABZCYFPUGVHI2DSMVQWIX3LMV43OSLTON2WKQ3PNVWWK3TUHMZDONJXHAYTCOBQG4>
> .
> You are receiving this because you were mentioned.Message ID:
> <SuperDataScience-Community-Projects/SDS-CP022-ai-travel-companion/pull/14/c2757811807
> @github.com>
>
